src/core_plugins/kibana/public/dashboard/grid/dashboard_grid.js

```diff
@@ -22,8 +30,10 @@ function ResponsiveGrid({ size, isViewMode, layout, onLayoutChange, children, ma
     'layout-view': isViewMode,
     'layout-edit': !isViewMode,
     'layout-maximized-panel': maximizedPanelId !== undefined,
+    'layout-with-margins': useMargins,
   });
 
+  const margins = useMargins ? 8 : 0;
```

Comment:
This is such a nitpicky comment, but how about uppercasing this "magic number"? `MARGINS`, since we're already doing that with `DASHBOARD_GRID_COLUMN_COUNT` and I can see needing to call this out as being special since we're specifying a visual style in JS (as opposed to CSS).